pointgrey_camera_driver/src/nodelet.cpp

```diff
@@ -514,10 +516,21 @@ class PointGreyCameraNodelet: public nodelet::Nodelet
             pub_->publish(wfov_image);
 
             // Publish the message using standard image transport
-            if(it_pub_.getNumSubscribers() > 0)
+            if(publish_camera_info_)
```

Comment:
It'd be cleaner to have a functor or function that you call like this: `publish(wfov_image->image);`

That function can be set on the constructor depending on the `publish_camera_info_` param.